modules/ansible/templates-ansible/configure-os-for-sap/playbook-configure-os-for-sap.yml.tftpl

```diff
@@ -15,8 +15,7 @@
     sap_domain: ${sap_domain}
 
   roles:
-  - { role: ibm.power_linux_sap.powervs_prepare_rhel_sap, when: sap_solution is defined and ansible_distribution is match("RedHat*") }
-  - { role: ibm.power_linux_sap.powervs_prepare_sles_sap, when: sap_solution is defined and ansible_distribution is match("SLES*") }
+   - { role: ibm.power_linux_sap.powervs_configure_os_for_sap, when: sap_solution is defined }
 
   tasks:
   - include_role:
```

Comment:
Lines from 21 to 43 must be removed as v3.0.0 does it already in the role ibm.power_linux_sap.powervs_configure_os_for_sap,